package.json

```diff
@@ -179,6 +179,7 @@
     "default-passive-events": "^2.0.0",
     "dequeue": "^1.0.5",
     "diff_match_patch": "^0.1.1",
+    "esprima": "^4.0.1",
```

Comment:
Dependency vetting checklist:

* [ ] **Actively maintained based on recent commit history:** No. There have been [no commits for the past 3 years](https://github.com/jquery/esprima/commits/main/), and [an issue from 2 years ago asking whether the project is abandoned](https://github.com/jquery/esprima/issues/2122) has not been responded-to by any maintainers

... Dropped rest of checklist

Doesn't look like esprima is maintained anymore, so let's avoid adding it as a dependency

Reply:
Here are the things I check when considering new dependencies: https://github.com/oppia/oppia-web-developer-docs/issues/327

Reply:
@U8NWXD, thank you for bringing my attention this. I will resolve it and revert.

Reply:
@U8NWXD I have removed the dependency has requested. 
Kindly review. Thank you.

Reply:
@U8NWXD Kindly check my update. Thank you.

Reply:
@FaithAbiola sorry, I didn't see this before because I wasn't assigned to the PR. Please make sure to assign anyone you need something from, otherwise we won't see it (most reviewers get way too many GitHub notifications to stay on top of at-mentions)

Reply:
@U8NWXD Thank you for your response. Sincere apologies for my late response 🙏🏾